lib/main/STM32F7/Drivers/STM32F7xx_HAL_Driver/Src/stm32f7xx_hal.c

```diff
@@ -154,7 +154,7 @@ __IO uint32_t uwTick;
 HAL_StatusTypeDef HAL_Init(void)
 {
   /* Configure Flash prefetch and Instruction cache through ART accelerator */ 
-#if (ART_ACCLERATOR_ENABLE != 0)
+#if (ART_ACCLERATOR_ENABLE != 0U)
```

Comment:
This is not necessary, I thought I have removed it to not introduce any changes into `lib` folder